libs/barista-components/stacked-series-chart/README.md

```diff
@@ -40,21 +40,22 @@ follow the same order given by the developer
-| `visibleLabel`           | `boolean`                                                | true     | Visibility of series label.                                                                                                                                                                                           |
-| `visibleValueAxis`       | `boolean`                                                | true     | Visibility of value axis.                                                                                                                                                                                             |
-| `maxTrackSize`           | `number`                                                 | 16       | Maximum size of the track.                                                                                                                                                                                            |
+| Name                     | Type                                                     | Default    | Description                                                                                                                                                                                                           |
+| ------------------------ | -------------------------------------------------------- | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `mode`                   | `DtStackedSeriesChartMode`                               | `'bar'`    | Display mode.                                                                                                                                                                                                         |
+| `series`                 | `DtStackedSeriesChartSeries[]`                           | -          | Array of series with their nodes.                                                                                                                                                                                     |
+| `selectable`             | `boolean`                                                | false      | Allow selections to be made on chart                                                                                                                                                                                  |
+| `selected`               | `[DtStackedSeriesChartSeries, DtStackedSeriesChartNode]` | -          | Current selection [series, node]                                                                                                                                                                                      |
```

Comment:
This is now only valid if `selectionMode` is `node`, if it's `stack` the  second parameter is optional, and in fact we have a new type for that, so it would be better if we specified that in the docs